autofill/autofill-impl/src/main/java/com/duckduckgo/autofill/impl/ui/credential/management/AutofillSettingsViewModel.kt

```diff
@@ -144,6 +151,8 @@ class AutofillSettingsViewModel @Inject constructor(
 
     private var combineJob: Job? = null
 
+    private var currentUrl: String? = null
```

Comment:
it's a bit of a weird one because the value comes from the UI originally and not the ViewModel, but it's the ViewModel that needs it

so it could be added back into the view state and stored there. but it's not actually needed in the view again

Reply:
Discussed this... will wrap up this `currentUrl` and the two related values into a single object and store it in the view state.

Not all of the values are used in the view per se, but they are all closely coupled that they should be one unit.

@cmonfortep i'll make that adjustment in an upper branch